CMSgov/design-system
# [NO-TICKET] Updating snapshots to get nightly tests to pass

## Summary

- Nightly tests have been failing for a while.
- Pulled main and ran `yarn test:browser -u` - lots of snapshots updated
- Not sure how these didn't get updated elsewhere?

## How to test

1. Can a reviewer run `yarn test:browser -u` on their local copy of `main` and make sure the snapshots really do need to be updated and that I'm not mistaken?

## Checklist
